core/io/file_access_pack.cpp

```diff
@@ -248,14 +189,152 @@ bool PackedSourcePCK::try_open_pack(const String &p_path, bool p_replace_files,
 		f->get_buffer(md5, 16);
 		uint32_t flags = f->get_32();
 
-		PackedData::get_singleton()->add_path(p_path, path, ofs + p_offset, size, md5, this, p_replace_files, (flags & PACK_FILE_ENCRYPTED));
+		PackedData::PackedFile pf;
+		pf.encrypted = !!(flags & PACK_FILE_ENCRYPTED);
```

Comment:
```suggestion
		pf.encrypted = (flags & PACK_FILE_ENCRYPTED);
```
Not needed and hard to read syntax, as you can see we use plain conversation elsewhere in this file